src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int colorNumber = new Random().nextInt(Color.values().length);
```

Comment:
```suggestion
    Random random = new Random();

    public Color getRandomColor() {
        int colorNumber = random.nextInt(Color.values().length);
```
Random must be a class variable